dist/components/select/_select.scss

```diff
@@ -45,6 +47,37 @@ $select__triangle-position: $select__h-space;
 
     user-select: none;
 
+
+    // "Down" Triangle
+    // ---
+    //
+    // 1. Absolutely position select element and triangle relative to the parent .c-select
+    // 2. Vertical center the triangle
+
+    &::after {
+        @include triangle(
+            $type: 'triangle',
+            $direction: 'down',
+            $size: $select__triangle-size,
+            $color: $select__border-color
+        );
+
+        content: '';
+
+        position: absolute; // 1, 2
+        top: 0; // 2,
+        right: $select__triangle-position;
+        bottom: 0; // 2
+
+        margin: auto; // 2
+    }
+
+
+    // Select
+    // ---
+    //
+    // 1. Absolutely position select element and triangle relative to the parent .c-select
+
```

Comment:
You could argue for or against it in this case. In my eyes, in select's case, the arrow block is signifigantly smaller and more management in this chunk, so it's easier for it to stay here.

So basically, I could go either way in this case.